test/rpc/test_rpc_call.py

```diff
@@ -23,14 +23,15 @@ def test_call(deploy_info, latest_block_id):
         params={
             "request": {
                 "contract_address": rpc_felt(contract_address),
-                "entry_point_selector": hex(get_selector_from_name("get_balance")),
+                "entry_point_selector": rpc_felt(get_selector_from_name("get_balance")),
                 "calldata": [],
             },
             "block_id": latest_block_id,
         },
     )
     assert "error" not in resp
     result = resp["result"]
+    assert_valid_rpc_schema(result, "starknet_call")
```

Comment:
Isn't it validated by default? (by decorator I mean)

Reply:
Good catch, this is a leftover of when validation was supposed to be only done in tests. I'll remove that.